### PR TITLE
feat: v0.5 blog post

### PR DESCRIPTION
**Pull request by ospencer**

### CODE REVIEW

**Review by marcusroberts — APPROVED** [review]
Looks excellent!

File: index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 </head>
 <body>
   <section class="site-banner">
-    <span>Say hello to <a href="/blog/2021/09/08/new-release-grain-v0.4-cereal/" target="_blank" rel="noopener">Grain v0.4 Cereal</a>—new platforms & improved tooling</span>
+    <span>New release <a href="/blog/2022/06/06/new-release-grain-v0.5-durum/" target="_blank" rel="noopener">Grain v0.5 Durum</a>—faster to build, faster to run</span>
```
Comment (peblair) [inline index.html:45]: ❤️

File: src_blog/_posts/New-Release-Grain-v0.5-Durum.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Hey folks, today we’re absolutely thrilled to announce the release of Grain v0.5, Durum! This release marks an enormous step forward for the language and tooling, all thanks to the tremendous effort put in by the team and community to deliver a release that we're truly proud of. You can expect huge compiler performance improvements, better language ergonomics, and as always, better tooling. You can find the full [release notes](https://github.com/grain-lang/grain/releases/tag/grain-v0.5.0) on GitHub but we wanted to highlight some things we are excited about.
+
+## Language Features
+
+First up: the new language features included in this release.
+
+### Arbitrary precision math
+
+In Grain v0.5, you’ll find that number operations no longer overflow—you can create numbers as large as you like, only limited by the total memory of your Wasm host. Whether you need to do some data science or you just like really big numbers, Grain has your back.
```
Comment (peblair) [inline src_blog/_posts/New-Release-Grain-v0.5-Durum.md:19]: We technically don't have arbitrary precision floats; I'm worried someone will interpret this as we do. Thoughts?

**Review by peblair — APPROVED** [review]
Looks great! One small comment, but this is 🔥

**Review by marcusroberts — COMMENTED** [review]
Actually, two big problems.  The gigantic amount of work done by Oscar and Blaine isn't attributed!  You two should have your names in there.

File: src_blog/_posts/New-Release-Grain-v0.5-Durum.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+### New LSP
+
+Previously, Grain editor support was exclusive to Visual Studio Code due to a required shim layer between the extension and compiler. In this release of Grain, core team member [Marcus](https://twitter.com/marcusr) added a long-running language server directly into the `grain` command that speaks the language server protocol.
+
+This means that the newest version of the VSCode extension is exclusively a “language client” that launches `grain lsp`, and any other editor that supports the language server protocol will be able to integrate seamlessly. Let us know what editors you want to see support for!
```
Comment (marcusroberts) [inline src_blog/_posts/New-Release-Grain-v0.5-Durum.md:79]: We could add that Emacs support is almost done, or if I hurry up, say it is done.  Also the new LSP has a huge performance increase over the previous version, and produces higher quality results for hovers

File: src_blog/_posts/New-Release-Grain-v0.5-Durum.md (hunk 2 above)
Comment (ospencer) [reply src_blog/_posts/New-Release-Grain-v0.5-Durum.md:19]: Eh, I'm not too worried about it. If someone comes asking for `BigFloat` we can address it then. I'll add "integer" to clarify.

Comment (ospencer) [reply src_blog/_posts/New-Release-Grain-v0.5-Durum.md:19]:
```suggestion
In Grain v0.5, you’ll find that number operations no longer overflow—you can create integer `Number`s as large as you like, only limited by the total memory of your Wasm host. Whether you need to do some data science or you just like really big numbers, Grain has your back.
```

**Pull request merged**